### PR TITLE
Add option to LSIIR to return rms-value

### PR DESCRIPTION
**Pull request by mgrub**
This PR will add the option to return the `rms`-value that is already printed in the case of `verbose=True`. 

This addresses issue #288 .

### CODE REVIEW

**Comment by mgrub** [comment]
Note: I am currently unable to reproduce the failing pipeline on my local development machine.

File: src/PyDynamic/model_estimation/fit_filter.py [hunk 1]
```diff
@@ -411,12 +426,15 @@ def _compute_x(
...
     rms = np.sqrt(np.sum(residuals_real_imag**2) / (len(residuals_real_imag) // 2))
-    print(
-        f"{_get_first_public_caller()}: Calculation of filter coefficients finished. "
-        f"Final rms error = {rms}"
-    )
+    if not suppress_print:
+        print(
+            f"{_get_first_public_caller()}: Calculation of filter coefficients finished. "
+            f"Final rms error = {rms}"
+        )
```
Comment (BjoernLudwigPTB) [inline src/PyDynamic/model_estimation/fit_filter.py:437]: I guess we should streamline this implementation. It does not seem very obvious and clear to me, that we have to introduce another parameter here, when we could simply split the function.

File: src/PyDynamic/model_estimation/fit_filter.py [hunk 2]
```diff
@@ -268,16 +274,25 @@ def LSIIR(
...
+        rms = _compute_and_print_rms(residuals_real_imag, suppress_print=not verbose)
+
+    return_list = [b_res, a_res, final_tau]
 
+    Uab = None
     if _uncertainties_were_provided(UH):
         Uab = np.cov(as_and_bs, rowvar=False)
-        return b_res, a_res, final_tau, Uab
-    return b_res, a_res, final_tau, None
+    return_list.append(Uab)
```
[the remark this answers is not in the document]
Comment (BjoernLudwigPTB) [reply src/PyDynamic/model_estimation/fit_filter.py]: Meanwhile resolved.

File: src/PyDynamic/model_estimation/fit_filter.py (hunk 1 above)
Comment (BjoernLudwigPTB) [reply src/PyDynamic/model_estimation/fit_filter.py]: meanwhile resolved by splitting both tasks in  separate functions

**Comment by BjoernLudwigPTB** [comment]
> Note: I am currently unable to reproduce the failing pipeline on my local development machine.

This was definitely not your fault. The tests can still and will probably always be improved, as long as we randomize the inputs and do not implement any proven error bounds but find our bounds via trial and error. Thanks for your effort and patience.

**Pull request merged**